domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/HowMuchPaid.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.cmc.domain.models.response;
+
+import org.hibernate.validator.constraints.NotBlank;
+import uk.gov.hmcts.cmc.domain.constraints.DateNotInTheFuture;
+import uk.gov.hmcts.cmc.domain.constraints.Money;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.Objects;
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class HowMuchPaid {
+
+    @NotNull
```

Comment:
I would prefer if the annotations are sorted in length order, it looks nicer IMO
i.e.
```
@Money
@NotNull
@DecimalMin
```
REC